src/map/Mapbox.ts

```diff
@@ -83,6 +88,119 @@ export default class Mapbox {
         this.drawSelectedPath(selectedPath)
     }
 
+    showPathDetails(selectedPath: Path) {
+        if (selectedPath.points.coordinates.length === 0)
+            return;
+        if (!this.map.hasControl(this.heightgraph))
+            this.map.addControl(this.heightgraph, 'bottom-right');
+        const elevation = this.createFeatureCollection(
+            'Elevation [m]',
+            [this.createFeature(selectedPath.points.coordinates, 'elevation')]
+        );
+        const pathDetails = Object.entries(selectedPath.details).map(([detailName, details]) => {
+            const points = selectedPath.points.coordinates;
+            const features = [];
+            for (let i = 0; i < details.length; i++) {
+                let [from, to, value] = details[i];
+                if (typeof value === 'undefined' || value === null)
+                    value = 'Undefined'
+                features.push(this.createFeature(points.slice(from, to + 1), value));
+            }
+            return this.createFeatureCollection(detailName, features);
+        });
+        const mappings: any = {};
+        mappings['Elevation [m]'] = function () {
+            return {text: 'Elevation [m]', color: '#27ce49'}
+        };
+        Object.entries(selectedPath.details).forEach(([detailName, details]) => {
+            mappings[detailName] = this.createColorMapping(details);
+        });
+        this.heightgraph.setData([elevation, ...pathDetails], mappings);
+    }
+
+    createFeature(coordinates: number[][], attributeType: number | string) {
```

Comment:
This method should be private. Like the other helper methods as well. I guess they could also be static since no state is used here.